t/t3433-rebase-options-compatibility.sh

```diff
@@ -84,4 +84,22 @@ test_expect_success '--committer-date-is-author-date works with interactive back
 	test_cmp authortime committertime
 '
 
+# Checking for +0000 in author time is enough since default
```

Comment:
That doesn't sound quite right.  The default time zone when committing should always be the timezone that's set up where the tests are run.  If we alwayse get UTC in the tests here, that's probably and indication that the code doesn't deal with timezones correctly.

And we don't set the timezone to UTC in the tests anywhere, as far as I can see.

Reply:
> The default time zone when committing should always be the timezone that's set up where the tests are run.

This is correct. But ...

> And we don't set the timezone to UTC in the tests anywhere, as far as I can see.

... we're actually setting timezone in [`test-lib.sh`](https://github.com/git/git/blob/v2.22.0/t/test-lib.sh#L368).

Reply:
> ... we're actually setting timezone in test-lib.sh.

Ah fair enough, I missed that. Thanks for the pointer.  I tried to `cat-file` a commit in t1010 to test this, but only now realized that we're using `test_tick` there, which confusingly sets the timezone to -0700.    So we're all good here, sorry for the noise.